build.gradle

```diff
@@ -1044,13 +1044,6 @@ final sharedPublicationConfiguration(publication) {
                 organization = 'University of Washington'
                 organizationUrl = 'https://www.cs.washington.edu/'
             }
-            developer {
-                id = 'wmdietl'
```

Comment:
Strange correction.

Reply:
You said last August that you no longer wanted to be a developer/maintainer.  This reflects that and takes you off the hook for maintenance that you do not want to do.

Reply:
That's not how that happened, but OK.